src/test/java/domain/ScoreBoardTest.java

```diff
@@ -0,0 +1,346 @@
+package domain;
+
+import domain.ScoreResult.BattleResult;
+import domain.ScoreResult.BattleResults;
+import domain.ScoreResult.ScoreBoard;
+import domain.card.GameCardDeck;
+import domain.participant.Dealer;
+import domain.participant.Participant;
+import domain.participant.Participants;
+import domain.participant.Player;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ScoreBoardTest {
+
+    @DisplayName("딜러 2패, 플레이어 1승, 플레이어 1승")
+    @Test
+    void 스코어보드_계산_확인1() {
+        //given
+        Player player1 = new Player("우가");
+        Player player2 = new Player("히스타");
+        Dealer dealer = new Dealer();
+
+        List<Participant> originParticipants = new ArrayList<>();
+        originParticipants.add(player1);
+        originParticipants.add(player2);
+        originParticipants.add(dealer);
+
+        Participants participants = new Participants(originParticipants);
+
+        GameCardDeck gameCardDeck = GameCardDeck.generateFullPlayingCard();
+        participants.drawTwoCards(gameCardDeck);
+        participants.drawTwoCards(gameCardDeck);
+
+        ScoreBoard scoreBoard = new ScoreBoard(participants);
+
+        //when
+        scoreBoard.calculateScoreBoard();
+
+        Map<Participant, BattleResults> battleResultsMap = scoreBoard.getScoreBoard();
+
+        //then
+        org.junit.jupiter.api.Assertions.assertAll(
+                () -> Assertions.assertThat(battleResultsMap.get(player1).getResults().get(BattleResult.WIN)).isEqualTo(1),
+                () -> Assertions.assertThat(battleResultsMap.get(player2).getResults().get(BattleResult.WIN)).isEqualTo(1),
+                () -> Assertions.assertThat(battleResultsMap.get(dealer).getResults().get(BattleResult.LOSE)).isEqualTo(2)
+        );
```

Comment:
```suggestion
    void 스코어보드_계산_확인1() {
        Participants participants = new Participants(List.of(new Player("우가"), new Player("히스타"), new Dealer()));

        GameCardDeck gameCardDeck = GameCardDeck.generateFullPlayingCard();
        participants.drawTwoCards(gameCardDeck);
        participants.drawTwoCards(gameCardDeck);

        ScoreBoard scoreBoard = new ScoreBoard(participants);

        //when
        scoreBoard.calculateScoreBoard();
        Map<Participant, BattleResults> battleResultsMap = scoreBoard.getScoreBoard();

        assertThat(battleResultsMap)
                .extractingFromEntries(
                        entry -> entry.getKey().getNickname(),
                        entry -> entry.getValue().getResults()
                )
                .contains(
                        Assertions.tuple("우가", Map.of(BattleResult.WIN, 1)),
                        Assertions.tuple("히스타", Map.of(BattleResult.WIN, 1)),
                        Assertions.tuple("딜러", Map.of(BattleResult.LOSE, 2))
                );
```

given절이 조금 복잡한 거 같네요 ㅠㅠ 이렇게 검증문을 수정하면 Player 객체를 유지할 필요없이 테스트를 작성할 수 있을 것 같습니다.